docs/index.md

```diff
@@ -9,7 +9,7 @@ more.
 
 [Installation](docs/getting-started/)
 
-## Example usage
+**Example usage**
```

Comment:
http://sequelize.readthedocs.org/en/latest/ it doesn't have one?
Headlines read way better.